text/0000-project-groups.md

```diff
@@ -0,0 +1,385 @@
+happen in the initial RFC thread. This leads to a lot of discussion in the top
+of the thread and that has can become completely irrelevant to the current
+iteration.
+
+This process has also been unsuitable to describe features that can take
+multiple years to develop and will become multiple RFCs over the course of its
+design process. Some examples of of this are the "`impl Trait`" and "macros
+2.0" features, where the goals has shifted a lot from the initial RFCs, and it
+can be hard to know their current status.
+
+### Project Group Creation
+
+Since project groups are approved by their relevant parent team over the core
+team, it's up to each team decide their specific requirements. However we
+recommend using the [application checklist](#application-checklist) as the basis
+for process and if needed adding any extra requirements.
+
+Process around project group membership is up to the shepherd's discretion.
+Typically, people who are productively contributing to the project group for
+some time will be added as members. It is not required for a project group to
+have alot of members though, some project groups may only have one or
+two members.
+
+### Project Group Evaluation
+
+Parent teams should add checking in with their project groups as part of their
+regular triage.
+
+[ffi unwind]: https://github.com/rust-lang/project-ffi-unwind
+[inline asm]: https://github.com/rust-lang/project-inline-asm
+[safe transmute]: https://github.com/rust-lang/project-safe-transmute
+
+## Community Groups
+
+Community Groups are groups of individuals who want to work and collaborate on a
+specific topic, without going through the formal process of a working group.
+Community groups are encouraged to adopt the structure of working or projects
+groups laid, but are free create and experiment with their own structure. As
+such community groups are not officially endorsed by The Rust Programming
+Language Organisation.
+
+## Archival
+
+The archival process of a group involves communicating the wind down to the
+community and the archival or transfer of ownership of the relevant projects.
+As well archiving any chat channels hosted by the Rust project.
+
+### Retrospectives
+
+While this RFC attempts to address some of the current organisational problems
+within the organisation, it also doesn't believe that this RFC will be a panacea
+to those problems or that we won't encounter more in the future. As part of
+that, we'd also like to introduce performing retrospectives with groups, once
+significant time has past or the group has been finished it's project.
+
+This would involve a discussion between the members of the group, and ideally
+their parent team and the Governance working group. The retrospective should
+produce a public blog post on the Inside Rust blog, however any feedback a
+member has that they would want to keep private would be omitted.
+
+The blog post should try to cover the output of the group, such as RFCs or
```

Comment:
I think the format used for documenting the output should be left to each team, and it could vary depending on the work. For example, I think a report in the form of https://github.com/rust-lang/rust/pull/67712 would be a perfectly good artifact. (This has the advantage that GitHub brings wrt. being able to link to everywhere with backlinks and whatnot.)

Reply:
@Centril I would push back on that. While the content of that report is good, its discoverability is quite poor. There's no indication that the PR contains a report, and even if it did the report will quickly be lost in the sea of all of the other activity on the repo. The point of having a blog post is so that reports like that are more easily found by people who don't have the history or experience to know what to look for.

Reply:
More discussion: https://discordapp.com/channels/442252698964721669/557273736118796289/669570273850425405.

Reply:
To summarise the discussion from discord. There was confusion over whether all output from a group be in blog posts. That is not the case, only the retrospective would be made a blog post. Of course if groups want to make more blog posts they can.

Reply:
I agree with @XAMPPRocky that blog posts are much more visible than PRs and the like. I also think that the content of a "retrospective blog post" might be quite different from a stabilization report, since ideally it would include observations on process as well. 

Regardless, I also think "cross-posting" is a thing. =) That is, it would be ok to post a blog post that links to a PR (and perhaps provides some additional material).

Reply:
(Remember that the most visible blog post links to a stabilization PR. ^^)

I think I'm happy with an internal requirement in the language team to have stabilization reports on GitHub and then retrospectives can come later, cross-posting to the report. Other teams may do as they wish. :)

Reply:
It seems like we can resolve this conversation, in that case? Are there edits to make, @XAMPPRocky ?

Reply:
I don't think so. Resolving.